e2e_samples/parking_sensors_synapse/devops/azure-pipelines-cd-release.yml

```diff
@@ -52,22 +56,26 @@ stages:
       serviceConnection: 'mdwdops-serviceconnection-stg'
   # Integration tests
   - template: templates/jobs/integration-tests-job.yml
-
-- stage: deploy_to_prod
-  displayName: 'Deploy to Prod'
-  variables:
-  - group: mdwdops-release-prod
-  - group: mdwdops-secrets-prod
-  jobs: 
-  - template: templates/jobs/deploy-dedicated-sql-pool-job.yml
+  - template: templates/jobs/deploy-sql-serverless-job.yml
     parameters:
-      environmentName: 'PROD'
-      serviceConnection: 'mdwdops-serviceconnection-prod'
-  - template: templates/jobs/deploy-wheel-job.yml
-    parameters:
-      environmentName: 'PROD'
-      serviceConnection: 'mdwdops-serviceconnection-prod'
-  - template: templates/jobs/deploy-synapse-workspace-job.yml
-    parameters:
-      environmentName: 'PROD'
-      serviceConnection: 'mdwdops-serviceconnection-prod'
\ No newline at end of file
+      environmentName: 'STG'
+      serviceConnection: 'mdwdops-serviceconnection-stg'
+
+# - stage: deploy_to_prod
```

Comment:
Need to uncomment the deploy to prod section, and add the `templates/jobs/deploy-sql-serverless-job.yml`